src/Stratis.Bitcoin.Features.RPC/Models/BlockchainInfoModel.cs

```diff
@@ -78,5 +79,69 @@ public class BlockchainInfoModel
 
         [JsonProperty(PropertyName = "pruned")]
         public bool IsPruned { get; set; }
+
+        [JsonProperty(PropertyName = "softforks")]
+        public List<SoftForks> SoftForks { get; set; }
+
+        [JsonProperty(PropertyName = "bip9_softforks")]
+        public Dictionary<string, SoftForksBip9> SoftForksBip9 { get; set; }
+    }
+
+    public class SoftForks
+    {
+        [JsonProperty(PropertyName = "id")]
+        public string Id { get; set; }
+
+        [JsonProperty(PropertyName = "version")]
+        public int Version { get; set; }
+
+        [JsonProperty(PropertyName = "reject")]
+        public SoftForksStatus Status { get; set; }
+    }
+
+    public class SoftForksStatus
+    {
+        [JsonProperty(PropertyName = "status")]
+        public bool Status { get; set; }
+    }
+
+    public class SoftForksBip9
+    {
+        [JsonProperty(PropertyName = "status")]
+        public string Status { get; set; }
+
+        [JsonProperty(PropertyName = "startTime")]
```

Comment:
@dangershony 👆

Reply:
ah good spot I missed this one I will fix

Reply:
Actually this follows bitcoin core.
https://github.com/bitcoin/bitcoin/blob/master/src/rpc/blockchain.cpp#L1162

🤔

Reply:
Ok, in that instance other parameters should be changed. It is confusing to use different casing convention for different properties, i.e. snake_case, lowercase, kebab-case, cameCase etc. It should be consistent. 
![image](https://user-images.githubusercontent.com/6656756/64423586-3fa1be00-d0ea-11e9-832a-736c21eb5096.png)

Reply:
Imagine you are setting up a client to parse the response. You would typically setup a deserialization convention for parameters. In this case, it is all different.

Reply:
There is a mixture of cases in the bitcoin RPC itself
![image](https://user-images.githubusercontent.com/7487930/64424402-e1042200-d0a0-11e9-9517-db69ffaf7f28.png)

I understand your point but I'd stay consistent with bitcoin core over the standard

Reply:
https://github.com/bitcoin/bitcoin/pull/16817

Reply:
This is so messy. I guess we have to go with the mess

Reply:
My hunch for this is that the word `starttime` is harder to read then `startTime` and because core already mixed cases its no big deal to make it more readable.

But yeah Id actually prefer snake case personally